package.json

```diff
@@ -57,13 +57,13 @@
     "jsdoc": "^3.6.6",
     "jsdoc-template": "braintree/jsdoc-template#3.2.0",
     "mkdirp": "^1.0.4",
-    "mocha": "^8.3.0",
+    "mocha": "^8.3.2",
     "npm-run-all": "^4.1.5",
     "serve-static": "^1.14.1",
     "through2": "^4.0.2",
     "vinyl": "^2.2.1",
     "vinyl-source-stream": "^2.0.0",
-    "watchify": "^3.11.1",
+    "watchify": "^4.0.0",
```

Comment:
This is the only dependency we had to make a major version change to, but the only breaking change is dropping support for Node.js versions older than 8.10